docs/DatabaseCatalogAPI.yaml

```diff
@@ -831,7 +818,7 @@ components:
         type: integer
         minimum: 0
         default: 0
-        example: 10
+        example: 0
```

Comment:
The error message for when I enter. negtive number is still:
`Value must be greater than 0`
Is there any way it could say something like:
`Value must be equal or greater than 0`?

Reply:
Spent a bit of time looking into it and seems like the default error message is hardcoded in the Swagger javascript somewhere. There is an `excludeMininum` option for the numbers type, but its value doesn't affect the message. 

It would take some effort to change this default error message (probably easier to not use the built-in validation and validate on the server side instead, but this doesn't sound ideal). Given that the default value and the example value are both 0 (and submitting a request with offset=0 doesn't produce a bad response), maybe users can infer that 0 is accepted. If we want, we can start a separate issue to look into updating the default error messages?

Reply:
Yes, a separate issue is good. It's really a minor problem.